src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import { Comments } from '../../types/Comments';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+
+type Props = {
+  comments: Comments[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(curr => {
```

Comment:
```suggestion
    {comments.map(comment => {
```
use better variable names